cookbooks/bcpc-hadoop/recipes/hive_hcatalog.rb

```diff
@@ -53,15 +53,6 @@
   user 'hdfs'
 end
 
-bash 'create-hive-scratch' do
-  code <<-EOH
-  hdfs dfs -mkdir -p #{node['bcpc']['hadoop']['hive']['scratch']['dir']}
-  hdfs dfs -chmod 1777 #{node['bcpc']['hadoop']['hive']['scratch']['dir']}
-  hdfs dfs -chown hive:hdfs #{node['bcpc']['hadoop']['hive']['scratch']['dir']}
-  EOH
-  user 'hdfs'
-end
```

Comment:
We shouldn't need hive to own something under the user's directory right @vt0r ?